deployment/helm/README.md

```diff
@@ -315,6 +315,11 @@ deployment:
       maxCount: "3"  # Max number of sessions per user.
       minEphemeralStorage: "20Gi"   # The initial requested amount of ephemeral (local) storage.  Does NOT apply to Desktop sessions.
       maxEphemeralStorage: "200Gi"  # The maximum amount of ephemeral (local) storage to allow a Session to extend to.  Does NOT apply to Desktop sessions.
+      
+      # Optionally setup a separate host for User Sessions for Skaha to redirect to.  The HTTPS scheme is assumed.  Defaults to the Skaha hostname (.Values.deployment.hostname).
+      # Example:
```

Comment:
Could this somehow also help with local install of the `science-platform` e.g., if you are running the Helm Chart of the Science Platform, you can point this to localhost?

Reply:
If `https://localhost` was setup properly, then it one could use `hostname: localhost`, yes.